src/app/teams/teams-view.component.ts

```diff
@@ -135,9 +139,6 @@ export class TeamsViewComponent implements OnInit, OnDestroy {
           onNext: () => {
             this.leaveDialog.close();
             const msg = 'left';
-            if (this.team.status === 'archived') {
```

Comment:
this section is there to redirect after leave button is trigger on team view.

Reply:
Thank you for the feedback. Will make changes.

Reply:
@lmmrssa - The code was added back to that section.